plugins/scaffolder/src/routes.ts

```diff
@@ -28,12 +28,22 @@ export const rootRouteRef = createRouteRef({
   id: 'scaffolder',
 });
 
+export const nextRouteRef = createRouteRef({
```

Comment:
Created a separate rootRouteRef for the next stuff so we can run it together under /next instead.

Reply:
Will this mean a future migration effort away from the next stuff? Or will this just stay "hidden" until we feel like flipping the switch on everyone? Could it have been enabled with a feature flag only for now? (and then both the old and the new routes would live next to each other)

Reply:
I think it's just going to run in parallel and then eventually it will replace the existing one, and we will drop the `/next` part.